.ruby-version

```diff
@@ -1 +1 @@
-1.9.3
+2.2.4
```

Comment:
I think dropping the file would make the most sense. Even if you did want to specify a version I think specifying it in the Gemfile makes more sense than in this file